### ISSUE TITLE
GLFW 3.3.1 not workin

### ISSUE THREAD

**Issue by hoechenberger** [psychopy/psychopy#2745]
See https://github.com/psychopy/psychopy/pull/2742#issuecomment-571545949

GLFW 3.3 is working fine.
@mdcutone I guess this one is for you :)

**Comment by mdcutone** [comment]
Seems to be working on my end with the updated version of pyGLFW. I have version 3.3.1 of the GLFW dll on my PATH and pyGLFW 1.9.

**Comment by hoechenberger** [comment]
Interesting, maybe it’s an issue that only surfaces on Linux or Travis then?

**Comment by hoechenberger** [comment]
Oh I see now we are still using 1.8 on Travis. Will look into this.

**Comment by mdcutone** [comment]
It might be a Linux specific thing.

**Comment by hoechenberger** [comment]
> It might be a Linux specific thing.

Yes, but it's odd it only started occurring with `glfw` 3.3.1, which -- according to the [changelog](https://www.glfw.org/changelog.html) -- only fixes some bugs found in 3.3. :(

**Comment by hoechenberger** [comment]
Seems to be a bug in `glfw` 3.3.1 itself: https://github.com/glfw/glfw/issues/1620

**Comment by mdcutone** [comment]
Looks like it was fixed, but we wont see it until the next release cycle.
